Kbs.Data/BoatType/BoatTypeRepository.cs

```diff
@@ -2,16 +2,49 @@
 using Kbs.Business.BoatType;
 using Kbs.Business.Reservation;
 using Microsoft.Data.SqlClient;
+using System.Collections.ObjectModel;
+using System.Windows.Input;
 
 namespace Kbs.Data.BoatType;
 
 public class BoatTypeRepository : IBoatTypeRepository
 {
     private readonly SqlConnection _connection = new(DatabaseConstants.ConnectionString);
+
     public List<BoatTypeEntity> GetAll()
     {
         return _connection.Query<BoatTypeEntity>("SELECT * FROM Boattype").ToList();
     }
+    public void Delete(BoatTypeEntity Boattype)
+    {     
+        /*
+        if (Boattype.BoattypeID == 0 || Boattype.BoattypeID == null) return;
+        _connection.Execute("DELETE FROM Boattype WHERE BoatTypeID = @BoatTypeID", new { BoatTypeID = Boattype.BoattypeID });
```

Comment:
waarom heb je dit stukje code niet gebruikt? Het lijkt me niet dat een boattypeid null of 0 kan zijn bij het selecteren, dus er hoeven toch geen foutmeldingen te komen?

Reply:
Was tijdelijke oplossing vanwege foutmelding. object == null -> nu opgelost zie laatste pull req